project/Build.scala

```diff
@@ -340,11 +285,11 @@ object Build {
         )
       }.evaluated,
 
-      filterTest := Def.inputTaskDyn {
+      vulpix := Def.inputTaskDyn {
         val args: Seq[String] = spaceDelimited("<arg>").parsed
-        testOptions := Seq()
+        val realArgs = if (args.nonEmpty) args else Seq(".*")
```

Comment:
Please, just change the filter to contain the regexp you pass as argument instead of being equal to it :).